nasa/harmony
# Harmony 1493 - A page limit input in the Workflow UI

## Jira Issue ID
HARMONY-1493

## Description
This ticket adds a page limit input to the workflow user interface on both of the jobs and job details page. It also sets the default page limit for the job details page to 100 work items (previously 1000).

<img width="1779" alt="Screenshot 2023-07-03 at 2 48 10 PM" src="https://github.com/nasa/harmony/assets/34752045/7d258cb6-e199-4424-b1e2-64bc0eb7c84e">

## Local Test Steps
- set the page limit via the page limit input and click apply
- check that the appropriate number of records is returned
- also check that `limit=` is set properly in the query string of the URL

## PR Acceptance Checklist
